components/quizzes/numberedcircles.tsx

```diff
@@ -3,9 +3,8 @@ import React from "react";
 const NumberedCircles = () => {
   return (
     <div className="relative grid grid-cols-1 gap-y-8 justify-center items-center">
-      {" "}
```

Comment:
all your quiz components should use camelCase

Reply:
understood, fixed.